src/panels/lovelace/editor/config-elements/hui-entities-card-editor.ts

```diff
@@ -20,6 +22,24 @@ import "../../components/hui-entity-editor";
 import "../../../../components/ha-card";
 import "../../../../components/ha-icon";
 
+const entitiesConfigStruct = struct.union([
+  {
+    entity: "string",
```

Comment:
Should we add a superstruct that validates entity ID and icon to be valid format?

- entity ID: should contain a `.`
- icon: should contain `:`

Reply:
yeah we should! 👍